rust-ceramic/src/license.md

```diff
@@ -0,0 +1 @@
+# License
```

Comment:
Is there supposed to be more content here?

Reply:
Should be, but removed for now.